.github/workflows/main.yml

```diff
@@ -0,0 +1,19 @@
+name: sync-snippet
+
+on:
+  schedule:
+    - cron:  '0 */24 * * *' # every 24 hours
```

Comment:
nit: These should be equivalent but I find fixing the hour to be slightly more readable
```suggestion
    - cron:  '0 0 * * *' # every 24 hours
```